tests/conftest.py

```diff
@@ -41,3 +43,43 @@ def db_with_one_user(db_empty):
     session.add(user)
     session.commit()
     return session
+
+
+@pytest.fixture
+def db_with_one_user_and_one_topic(db_with_one_user):
+    session = db_with_one_user
+    topic = Topic.create_topic(title="test title", description="test desc", author_id=1)
+    topic.id = 1
+    session.add(topic)
+    session.commit()
+    return session
+
+
+@pytest.fixture
+def db_with_three_users(db_with_one_user):
+    session = db_with_one_user
+    user_two = User(id=2, username="test2")
+    user_three = User(id=3, username="test3")
+    user_two.password_hash = hashlib.sha256("testtest2".encode()).hexdigest()
+    user_three.password_hash = hashlib.sha256("testtest3".encode()).hexdigest()
```

Comment:
It is better to group lines by created objects instead of actions made upon them. In other words, now we have:
```python
# call constructor 1
# call constructor 2

# set password hash 1
# set password hash 2

# add to session 1
# add to session 2
```

But if we regroup lines like this it will make our readability much better and allow us to maintain this code for a longer time:
```python
# call constructor 1
# set password hash 1
# add to session 1

# call constructor 2
# set password hash 2
# add to session 2
```

The correct way of this can be seen in your `db_with_three_users_and_tree_topics` fixture.